plugins/plugins.go

```diff
@@ -153,6 +153,7 @@ type Manager struct {
 	initialized                  bool
 	interQueryBuiltinCacheConfig *cache.Config
 	gracefulShutdownPeriod       int
+	reconfigureListeners         []func(*config.Config, *cache.Config)
```

Comment:
What about something like `registeredCacheTriggers   []func(*cache.Config)` ?

Reply:
I'm good with that if we're going to eliminate `*config.Config`.